demo/csi/aws/ebs/example-volume.hcl

```diff
@@ -0,0 +1,10 @@
+# terraform output mytestebsvol >example-volume.hcl
```

Comment:
If we intend for these example.hcl files to be generated by Terraform, we shouldn't also commit them to the repo. (I'd even drop a `.gitignore` file in this directory to be sure folks don't commit the volume)

Reply:
I think it's useful to have a simple example file that someone can try to understand without trying to understand Terraform at the same time. I'll change the comments so it's clear that this is a separate example from the Terraform examples.